code/__HELPERS/cmp.dm

```diff
@@ -81,3 +81,21 @@ GLOBAL_VAR_INIT(cmp_field, "name")
 
 /proc/cmp_advdisease_resistance_asc(datum/disease/advance/A, datum/disease/advance/B)
 	return A.totalResistance() - B.totalResistance()
+
+/proc/cmp_quirk_asc(datum/quirk/A, datum/quirk/B)
```

Comment:
hmm?

Reply:
Relying on type ordering will be very difficult to do if the file with the types is split or rearranged.

Reply:
yep I see it now